src/components/canrawsender/canrawsender_p.h

```diff
@@ -30,14 +31,22 @@ class CanRawSenderPrivate : public QWidget {
         connect(ui->pbDockUndock, &QPushButton::pressed, this, &CanRawSenderPrivate::dockUndock);
     }
 
-    ~CanRawSenderPrivate()
+    ~CanRawSenderPrivate() {}
```

Comment:
Possibly virtual, `~CanRawSenderPrivate() = default` to be put in the header file.